cmake/projects/NASM/schemes/url_sha1_nasm_windows.cmake.in

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2017, Zhuhao Wang
```

Comment:
Please take a look at this scheme:
* https://github.com/ruslo/hunter/blob/master/cmake/schemes/url_sha1_unpack_install.cmake.in